build-info-extractor/src/main/java/org/jfrog/build/extractor/scan/DependencyTree.java

```diff
@@ -82,6 +115,11 @@ public int getIssueCount() {
         return children != null ? children : new Vector<>();
     }
 
+    @JsonProperty(value = "nodes")
+    public List<DependencyTree> getNodes() {
+        return children;
+    }
```

Comment:
Any reason for returning list instead of vector?

Reply:
Why not? is more generic and used mainly for serialization. 
How returning a Vector is better?

Reply:
Sorry I didn't know that vector is a list. In that case, please add a suppression above the method:
```java
@SuppressWarnings({"unchecked", "unused"})
```